packages/authgear-web/src/container.ts

```diff
@@ -331,14 +331,7 @@ export class WebContainer {
       throw new AuthgearError("missing client ID");
     }
     const config = await this.baseContainer.apiClient._fetchOIDCConfiguration();
-
-    const { id_token } = await this.baseContainer.apiClient._oidcTokenRequest({
-      grant_type: "urn:authgear:params:oauth:grant-type:id-token",
```

Comment:
Do you think we should also remove this custom grant type support from the server?